packages/frontend/src/components/wallet-migration/WalletMigration.jsx

```diff
@@ -71,13 +81,17 @@ const WalletMigration = ({ open, onClose }) => {
     }, [handleSetActiveView]);
 
     const onContinue = useCallback(async () => {
-        let url = '';
-        url = await encodeAccountsToURL(
-            availableAccounts,
-            state.migrationKey,
-            state.wallet
-        );
-        window.open(url, '_blank');
+        if (state.wallet.id === 'sender') {
+            handleSetActiveView(WALLET_MIGRATION_VIEWS.SENDER_MIGRATION_TYPE_SELECT);
+        } else {
+            let url = '';
+            url = await encodeAccountsToURL(
+                availableAccounts,
+                state.migrationKey,
+                state.wallet
+            );
+            window.open(url, '_blank');
+        }
```

Comment:
can we pull this logic out above, into a function and have something like:
```
switch (walletId) {
  case 'sender': {
    return ...;
  }

  default: {
    return ...;
  }
}
```

So that it is easier for other wallets to apply their custom actions as well.